doc/license.md

```diff
@@ -0,0 +1,18 @@
+# License
+
+The Data Observatory is a collection of various sources of data with varying licenses. We have worked hard to find you data that will work for the broades set of use-cases. Please still review the below terms for any dataset you use and please respect the rights of the owners of each dataset you use. We list below the third-party datasets that are used in the Data Observatory, along with the terms governing the use of each.
```

Comment:
Did you mean "broadest"? Are the "below terms" still coming? Or are you just listing the third-party datasets?